test_driver/mocks/utils/verify_dispatch_middleware.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:crowdleague/actions/redux_action.dart';
+import 'package:crowdleague/models/app/app_state.dart';
+import 'package:redux/redux.dart';
+
+/// [Middleware] that keeps track of actions that have been dispatched, for
+/// testing.
+///
+/// [actions] is a static/class member so that it can be accessed in the
+/// anonymous funciton that is created in the class initializer
+class VerifyDispatchMiddleware extends TypedMiddleware<AppState, ReduxAction> {
+  static List<ReduxAction> actions = [];
+  VerifyDispatchMiddleware()
+      : super((store, action, next) async {
+          next(action);
```

Comment:
do we want the next method taken out of here too?
i just noticed it

Reply:
let's just delete the whole thing!  I might make a clean up PR after this goes in, just to clean up the mock files I added messily, so I can remove it then :-)